uint32 offset
---
bool result
